datajunction-ui/src/app/pages/NodePage/ClientCodePopover.jsx

```diff
@@ -1,46 +1,90 @@
 import { Light as SyntaxHighlighter } from 'react-syntax-highlighter';
 import { useEffect, useRef, useState } from 'react';
-import { nightOwl } from 'react-syntax-highlighter/src/styles/hljs';
+import { nightOwl } from 'react-syntax-highlighter/dist/esm/styles/hljs';
 import PythonIcon from '../../icons/PythonIcon';
 
 export default function ClientCodePopover({ code }) {
-  const [codeAnchor, setCodeAnchor] = useState(false);
-  const ref = useRef(null);
+  const [showModal, setShowModal] = useState(false);
+  const modalRef = useRef(null);
 
   useEffect(() => {
-    const handleClickOutside = event => {
-      if (ref.current && !ref.current.contains(event.target)) {
-        setCodeAnchor(false);
+    const handleClickOutside = (event) => {
+      if (modalRef.current && !modalRef.current.contains(event.target)) {
+        setShowModal(false);
       }
     };
-    document.addEventListener('click', handleClickOutside, true);
+
+    if (showModal) {
+      document.addEventListener('mousedown', handleClickOutside);
+    } else {
+      document.removeEventListener('mousedown', handleClickOutside);
+    }
+
     return () => {
-      document.removeEventListener('click', handleClickOutside, true);
+      document.removeEventListener('mousedown', handleClickOutside);
     };
-  }, [setCodeAnchor]);
+  }, [showModal]);
 
   return (
     <>
-      <button
-        className="code-button"
-        aria-label="code-button"
-        tabIndex="0"
-        height="45px"
-        onClick={() => setCodeAnchor(!codeAnchor)}
-      >
-        <PythonIcon />
+      <button className="button-3" onClick={() => setShowModal(true)} style={{height: "2.5rem"}}>
+        <PythonIcon /> See Python
```

Comment:
Feel free to also remove this Python client code modal if more screen real estate is needed, or omit the words "See Python".